stable_baselines3/common/off_policy_algorithm.py

```diff
@@ -67,6 +67,7 @@ class OffPolicyAlgorithm(BaseAlgorithm):
     :param use_sde_at_warmup: (bool) Whether to use gSDE instead of uniform sampling
         during the warm up phase (before learning starts)
     :param sde_support: (bool) Whether the model support gSDE or not
+    :param n_step: (int) Number of steps to consider when computing the target Q-value
```

Comment:
minor (mostly my fault): docstring missing for the replay buffer class and kwargs